### PR TITLE
Add eslint check to test dir

### PR DESCRIPTION
**Pull request by mnibecker**
@go-oleg

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -7,7 +7,8 @@
     "env": {
         "browser": true,
         "node": true,
-        "es6": true
+        "es6": true,
+        "mocha": true
```
Comment (go-oleg) [inline .eslintrc]: you can add another `.eslintrc` file inside the `test/` dir to make this `mocha` thing only apply to files in that dir.

**Comment by go-oleg** [comment]
comment above, otherwise lgtm

File: .eslintrc (hunk 1 above)
Comment (mnibecker) [inline .eslintrc]: cool, will do

**Comment by mnibecker** [comment]
@go-oleg, done. Great tip!

**Pull request merged**